dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.38/V2_38_30__Add_attributefilter_column_to_trackedentityinstancefilter.sql

```diff
@@ -0,0 +1 @@
+alter table trackedentityinstancefilter add column if not exists attributevaluefilters jsonb default '[]'::jsonb;
```

Comment:
Duplicated script?
Or you just want to be really sure? :D

Reply:
haha. Actually Zubair "reserved" the 2.38.29 flyway version so I renamed my file. but for some reason, it got pushed as a new file. Deleted the 2.38.29. The new script is 2.38.30 :) Good catch!